src/main/java/com/flipkart/yak/commons/HBaseUtils.java

```diff
@@ -13,8 +13,13 @@
 public class HBaseUtils {
 
     public static List<RegionInfo> getRegionsAll(CompactionContext context, Admin admin) throws IOException {
-        TableName tableName= TableName.valueOf(context.getNameSpace()+":"+context.getTableName());
-        return admin.getRegions(tableName);
+        List<RegionInfo> allRegions = new ArrayList<>();
+        String[] tables = context.getTableName().split(",");
```

Comment:
You should add a new field for tablenames in context. Create a new function for getRegionsAllTables or something and thats how you keep it backward compatibility